foundations/html_css/html-foundations/html-boilerplate.md

```diff
@@ -177,7 +177,7 @@ This section contains questions for you to check your understanding of this less
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- Another option for opening your HTML pages in the browser is using the [live server extension](https://marketplace.visualstudio.com/items?itemName=ritwickdey.LiveServer) with VSCode. This will open your HTML document and automatically refresh it every time you save the document. However, we recommend not using this extension and instead doing it the old-fashioned way, by opening the page and refreshing the page manually in the browser for your first few HTML projects. This way, you can get used to that process and won't become reliant on extensions right away. One reason is that there may be subtle differences when using extensions. For example, live server will always use UTF-8 character encoding and not the charset value defined in your `<meta>` element. This could potentially hide some characters on your site as they will not be encoded the way you expect.
+- Another option for opening your HTML pages in the browser is using Microsoft's [live preview extension](https://marketplace.visualstudio.com/items?itemName=ms-vscode.live-server) with VSCode. This will open your HTML document and automatically refresh it every time you make an edit. However, we recommend not using this extension and instead doing it the old-fashioned way, by opening the page and refreshing the page manually in the browser for your first few HTML projects. This way, you can get used to that process and won't become reliant on extensions right away. One reason is that there may be subtle differences when using extensions. For example, live server will always use UTF-8 character encoding and not the charset value defined in your `<meta>` element. This could potentially hide some characters on your site as they will not be encoded the way you expect.
```

Comment:
@Falciron  @bycdiaz any opinions on the above?

Reply:
If we recommend not using it, my sense is that it might be better to not mention it here and instead in a future lesson.

Reply:
@bycdiaz Agreed. I can't remember the intent of including the link as an additional resource but saying not to use it, but I honestly don't think using the live server extension is all that bad for users to be doing. Manually refreshing a page every time you make a change can be really tedious and I don't think really offers any significant learning experience after the first time you have to do it.

What do you think about moving this content to the Recipes project, before the main assignment (or as the first step of the assignment)? Users would have to do things the "old fashioned way" for a bit before being introduced to something where it would be more handy.